src/graphql/types/object/usd-wallet.ts

```diff
@@ -42,6 +43,11 @@ const UsdWallet = GT.Object<Wallet>({
         return Math.floor(balanceCents)
       },
     },
+    pendingIncomingBalance: {
+      type: GT.NonNull(SignedAmount),
+      description: "An unconfirmed incoming onchain balance.",
+      resolve: () => notBtcWalletForQueryError,
```

Comment:
I'm wondering if we need this error? or if the same code that we have for BTC wallet would just work?

assuming we can't generate an address currently on usd wallet for onchain, this function would basically always return 0 which is also a valid result?

another option would be to simply not expose `pendingIncomingBalance`? (maybe we can't because there is some inheritance?)